scripts/celo/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env sh
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+# Exit on failure
+set -ex
+
+rm -rf /root/celo
+geth init --datadir /root /root/genesis.json
+
+geth --datadir /root --maxpeers 0 --light.maxpeers 0 --password /root/password.txt account import /root/key.txt
+#rm -rf /root/keystore
+#cp -r /root/keystore /root/.ethereum/
+
+
+
+exec geth --datadir /root \
```

Comment:
To be clear, the data dir should be /root/.ethereum. Furthermore, you don't need to specify a datadir because that is the default path ($HOME/.ethereum, $HOME is /root for the root user)